partition_table.go

```diff
@@ -194,43 +194,58 @@ func (p *PartitionTable) Close() error {
 func (p *PartitionTable) createStorage(ctx context.Context) (*storageProxy, error) {
 	var (
 		err  error
+		errs = new(multierr.Errors)
 		st   storage.Storage
 		done = make(chan struct{})
 	)
+
 	start := time.Now()
 	ticker := time.NewTicker(5 * time.Minute)
```

Comment:
`ticker` declaration and `defer ticker.Stop` should go inside logging goroutine since it only used there.
The logging is then stopped only through the `done` channel.

Reply:
Good point.